apps/frontend/src/libs/components/search/search.tsx

```diff
@@ -0,0 +1,40 @@
+import { Icon, Input } from "~/libs/components/components.js";
+import { debounce } from "~/libs/helpers/helpers.js";
+import { useAppForm, useEffect } from "~/libs/hooks/hooks.js";
+
+type Properties = {
+	onValueChange: (search: string) => void;
```

Comment:
Not sure, but maybe it will be okay, if we just call this `onChange`

Reply:
renamed